memecity.game/Game/Components/ItemComponent.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _ITEMCOMPONENT_H
+#define  _ITEMCOMPONENT_H
+#include <ECS.h>
+#include <iostream>
+#include <Engine\MultimediaManager.h>
+
+struct ItemComponent : public memecity::engine::ecs::Component
+{
+	std::string name, description;
+
+	int damage, defence;
```

Comment:
`defense`

Reply:
Zover ik weet is defence ook een correct wordt ?